server/datastore/mysql/migrations/tables/20240521143023_CreateTableMDMAssets_test.go

```diff
@@ -0,0 +1,60 @@
+package tables
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240521143023(t *testing.T) {
+	db := applyUpToPrev(t)
+	applyNext(t, db)
+	const (
+		insertStmt     = `INSERT INTO mdm_config_assets (name, value) VALUES (?, ?)`
+		selectStmt     = `SELECT * FROM mdm_config_assets WHERE name = ? AND deleted_at IS NULL`
```

Comment:
Maybe you used that deliberately for the test, but just in case, the idea would be to use `name = ? AND deletion_uuid = ''` in production code (so that it fully uses the unique index).